addon/addon-test-support/@ember/test-helpers/dom/-get-element.ts

```diff
@@ -32,7 +39,14 @@ function getElement(target: Target): Element | Document | null {
   } else if (target instanceof Window) {
     return target.document;
   } else {
-    throw new Error('Must use an element or a selector string');
```

Comment:
non-blocking, and maybe stylistic, but I really like using assert for type-narrowing and error-communication.

this bit of code would be:

```js
assert('Must use an element, sel...', descriptorData)

return resolveDOMElement(descriptorData);

```
- less nesting
- less branching
- we want an error anyway 

✨